test/integration_test.py

```diff
@@ -300,25 +332,25 @@ def test_indexing(self):
         @attr.s(auto_attribs=True, kw_only=True)
         class Catalog:
             name: CatalogName
-            notifications: Mapping[SourcedBundleFQID, JSON]
+            bundles: Set[SourcedBundleFQID]
+            notifications: List[JSON]
             random: Random = self.random
 
             @property
             def num_bundles(self):
-                return len(self.notifications)
-
-            @property
-            def bundle_fqids(self) -> AbstractSet[SourcedBundleFQID]:
-                return self.notifications.keys()
+                return len(self.bundles)
 
             def notifications_with_duplicates(self) -> List[JSON]:
-                num_duplicates = self.num_bundles // 2
-                notifications = list(self.notifications.values())
-                # Index some bundles again to test that we handle duplicate additions.
-                # Note: random.choices() may pick the same element multiple times so
-                # some notifications will end up being sent three or more times.
-                notifications.extend(self.random.choices(notifications, k=num_duplicates))
-                return notifications
+                # FIXME: Add back duplicates
+                #        https://github.com/DataBiosphere/azul/issues/3558
+                if False:
```

Comment:
Neither the ticket nor the comment explains why this has to be disabled.

Reply:
I'm not sure myself. Added back duplicates and it seems to work fine but I'm worried I'm missing something.

Reply:
As I recall, @hannes-ucsc had explicitly asked me to disable this, and keep things as simple as possible for a mvp.